StyleGuide.md

```diff
@@ -542,6 +543,92 @@ There is also a custom purple callout that uses the code `{{site.data.alerts.cal
+
+We list the general differences between CockroachDB and the SQL Standard/PostgreSQL on our [SQL Feature Support](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html) page, and we provide more details on the differences between CockroachDB and PostgreSQL on our [PostgreSQL Compatibility](https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html). All other instances of known, but possibly unexpected, database behavior are known as *known limitations*.
+
+Known limitations often have [associated GitHub issues in the `cockroach` repo](https://github.com/cockroachdb/cockroach/issues), meaning the limitation could be resolved one day (in theory...). *Not all known limitations have GitHub issues, and not all known limitations will be resolved.*
+
+The purpose of documenting a known limitations is to help our users know more about using our product safely and effectively.
+
+#### Where to find known limitations
+
+Known limitations are generally listed in two places:
+
+1. (More common) In the `cockroach` repo, as [open issues with the `docs-known-limitations` label, but *not* with the `docs-done` label](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+label%3Adocs-known-limitation+-label%3Adocs-done+is%3Aopen). Usually, engineers add these labels to issues in the weeks leading up to the release.
+
+1. (Less common) In the `docs` repo, as [open issues with the `T-known-limitation` label](https://github.com/cockroachdb/docs/issues?q=is%3Aopen+is%3Aissue+label%3AT-known-limitation).
+
+If you come across some behavior that you believe qualifies as a known limitation, first open an issue in the `cockroach` repo, get some engineering/PM feedback on the issue, and then add a `docs-known-limitations` label to an issue.
+
+#### When to document known limitations
+
+Documenting known limitations should happen in the [weeks leading up to a GA release](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/402718726/GA+Release+Checklist).
+
+*You can document a known limitation too early (usually in the event that the limitation is a bug that engineering finds the time to fix during the stability period leading up to a GA release).*
+
+#### Who documents known limitations
+
+In the past, the person assigned to known limitations is usually someone with extra bandwidth at the end of the GA release cycle. You might volunteer for this task, or your manager might assign it to you.
+
+#### Where to document known limitations
+
+Each known limitation should be documented on the [Known Limitations](https://www.cockroachlabs.com/docs/stable/known-limitations.html) page.
+
+If the limitation is related to a feature documented elsewhere on our docs site, the page that documents that feature should also document the limitation, under a dedicated "Known limitations" header. In this case, you usually want to create a file in `_includes/vX.X/known-limitations` and just include the file in both places.
```

Comment:
Suggest editing the last line about includes to be slightly more directive, maybe something like

"If the limitation appears on both the KL page and the feature page, you should use an include file located at X to avoid duplication if at all possible"

or something like that

Reply:
Done and done.